app/services/questionnaire_responses/monthly_screening_reports.rb

```diff
@@ -3,17 +3,17 @@ class QuestionnaireResponses::MonthlyScreeningReports
 
   def initialize(date = 1.month.ago)
     @month_date = date.beginning_of_month
-    @facility_reports = monthly_followup_and_registration
+    @facility_ids = Facility.select("id").pluck(:id)
     @questionnaire_id = latest_active_questionnaire_id(Questionnaire.questionnaire_types[:monthly_screening_reports])
   end
 
-  def pre_fill
-    @facility_reports.each do |facility_report|
-      unless monthly_screening_report_exists?(facility_report.facility_id)
+  def seed
```

Comment:
We're just seeding(initializing) forms instead of pre-filling data, because India doesn't have screening forms and it isn't feasible to pre-fill data in Ethiopia's forms due to new logic of advanced initialization.